Sources/ArcGISToolkit/Components/Augmented Reality/CalibrationView.swift

```diff
@@ -0,0 +1,170 @@
+                    }
+                    .onChange(of: headingSliderValue) { heading in
+                        guard headingTimer == nil else { return }
+                        // Create a timer which rotates the camera when fired.
+                        let timer = Timer(timeInterval: 0.1, repeats: true) { _ in
+                            rotateHeading(by: joystickHeadingDelta)
+                        }
+                        headingTimer = timer
+                        // Add the timer to the main run loop.
+                        RunLoop.main.add(timer, forMode: .default)
+                    }
+                    
+                    Button {
+                        let heading = viewModel.cameraController.originCamera.heading + 1
+                        updateHeading(to: heading)
+                    } label: {
+                        Image(systemName: "plus")
+                            .imageScale(.large)
+                    }
+                }
+                VStack {
+                    Text("Elevation: \(viewModel.calibrationElevation?.rounded(.towardZero) ?? viewModel.cameraController.originCamera.location.z?.rounded(.towardZero) ?? 0, format: .number)")
+                    HStack {
+                        Button {
+                            updateElevation(by: -1)
+                        } label: {
+                            Image(systemName: "minus")
+                                .imageScale(.large)
+                        }
+                        Slider(value: $elevationSliderValue, in: -20...20) { editingChanged in
+                            if !editingChanged {
+                                elevationTimer?.invalidate()
+                                elevationTimer = nil
+                                elevationSliderValue = 0.0
+                            }
+                        }
+                        .onChange(of: elevationSliderValue) { elevation in
+                            guard elevationTimer == nil else { return }
+                            // Create a timer which rotates the camera when fired.
+                            let timer = Timer(timeInterval: 0.1, repeats: true) { _ in
+                                updateElevation(by: joystickElevationDelta)
+                            }
+                            elevationTimer = timer
+                            // Add the timer to the main run loop.
+                            RunLoop.main.add(timer, forMode: .default)
+                        }
+                        Button {
+                            updateElevation(by: 1)
+                        } label: {
+                            Image(systemName: "plus")
+                                .imageScale(.large)
+                        }
+                    }
+                }
+            }
+            .onDisappear {
+                guard !viewModel.isCalibrating else { return }
+                viewModel.setBasemapOpacity(0)
```

Comment:
I'm seeing an error about dynamic member lookup on this line, but the app still compiles 🤔 Do you see an error here?

Reply:
I do not see this error on my machine.